spec/fixtures/README.md

```diff
@@ -0,0 +1,104 @@
+# One More Time
```

Comment:
Hilarious, but I don't think we need _this_ big of a file to test that it's working 😉.